packages/devp2p/src/dpt/server.ts

```diff
@@ -22,14 +22,14 @@ const verbose = createDebugLogger('verbose').enabled
 const VERSION = 0x04
 
 export class Server extends EventEmitter {
-  _dpt: DPT
-  _privateKey: Uint8Array
-  _timeout: number
-  _endpoint: PeerInfo
-  _requests: Map<string, any>
-  _requestsCache: LRUCache<string, Promise<any>>
-  _socket: DgramSocket | null
-  _debug: Debugger
+  protected _dpt: DPT
+  protected _privateKey: Uint8Array
+  protected _timeout: number
+  protected _endpoint: PeerInfo
+  protected _requests: Map<string, any>
+  protected _requestsCache: LRUCache<string, Promise<any>>
+  protected _socket: DgramSocket | null
+  private _debug: Debugger
```

Comment:
Likely not practically relevant and only a small nit, but I would think that it would make sense to just not distinguish between "even more internal" or so variables and just also make `_debug` protected, maybe someone wants to customize the devp2p package and add own debug messages, whatever.

Reply:
I was on the fence about this one, but since debuggers are usually initialized specific to a class, I opted for private.

Reply:
The `protected` version is also remaining in the class scope, so this is just a bit broader so that people can e.g. subclass the `Server` class, e.g. for a custom binance devp2p implementation (super random example) or whatever. And then on the subclass one might want to adjust some properties and it is just a bit unlucky if these are then not accessible. So at some point in time [tm] we generally decided to favor `protected` over `private`, since it (more or less) achieves the same thing and has fewer side effects.